util/config/validators/validate_ephemeral_builder.py

```diff
@@ -0,0 +1,48 @@
+from buildman.manager.executor import EC2Executor, KubernetesExecutor
+
+from util.config.validators import BaseValidator, ConfigValidationException
+
+
+class EphemeralBuilderValidator(BaseValidator):
+    name = "ephemeral-builder"
+
+    @classmethod
+    def validate(cls, validator_context):
+        config = validator_context.config
+
+        if not config.get("FEATURE_BUILD_SUPPORT", False):
+            return
+
+        if not config.get("BUILD_MANAGER"):
+            return
+
+        build_manager_config = config["BUILD_MANAGER"]
+        if build_manager_config[0] != "ephemeral":
+            return
+
+        if len(build_manager_config) != 2:
```

Comment:
Move this above to before checking for ephemeral.

Reply:
Why? We check if there are any elements above the ephemeral check.

Reply:
The above will raise an IndexError instead of a ConfigValidationException.

Reply:
`        if not config.get("BUILD_MANAGER"):` will catch it, but I'll change the order to make it nicer